### DISCUSSION TITLE
Having more option using interview function

### DISCUSSION THREAD

**Discussion by danje57** [monarc-project/MonarcAppFO#410] [Ideas]
Hi all,

I think it would be great to extend the Interview functionality.

As each risks should have a Risk Owner, it will be great having the Interview option available to document each risk to keep a record of all answer given by the risk owner during the interview for all risks.

As of today, I find this option useless as is (just my opinion)

Regards

**Comment by jerolomb** [comment]
Hello,

Thanks for the suggestion. For the moment, you can use the risk context on the risk sheet to add more infos on the related risk. 

Regards

**Reply by danje57** [reply]
This is what I do for the moment :)

However, I use Monarc since 1 week but I appreciate it, simplicity, options.... Asome tool!

**Comment by ruslanbaidan** [comment]
Hello,

Could you please describe your idea in more details.

I made a screenshot of an opened risk and interviews table (attached below) and there we have Department / People field. And the interview functionality is more general not about risks, but would be good to understand how do you see it exactly.

Thank you.

**Reply by danje57** [reply]
I don't see your sreenshot.

I think about a functionality that will be closed to the risk description (in the same screen). So you could have for example an interview Field where we can write annotation regarding the person interviewed.

For example, in my own case, I d'ont know all security controls in palce. So sometime I need to discuss with the risk owner to verify the control in place and to document, for traceability purpose the name of the people interviewed and its answer.

HAving this on each risk will be also a good feature for reporting purpose as we can have directly for each risk the discussion made with the interviewed person.

Regards

**Reply by ruslanbaidan** [reply]
I've attached the file now.
![InterviewTable](https://user-images.githubusercontent.com/3246171/159874451-84eb705f-d005-4507-8bb4-be0513373b74.png)

What I can see from your explanation, that you would like to see a new filed/relation with risk owner in the interview table. The risk normally can be specified in the Content field.
Could that work out?

Regards.

**Reply by danje57** [reply]
This is a good feature.

However, I think the feature can be improved if we have access to it directly near the risk discussed with the interviewed person.

For example:
![image](https://user-images.githubusercontent.com/16591723/160573219-79f7cba6-33e7-4c7b-abb5-2742c3dcba55.png)

In this Image, I want to discuss this risk with MR XYZ which is the owner of the risk.

I think, the functionality should be improved by using a button near  all risks, whoch call the interview functionality.
Than we can add discussion we have for this particular risk and so on...

![image](https://user-images.githubusercontent.com/16591723/160574611-65a2394f-0cff-4c02-a547-f07a098311ec.png)

Hope this image can help to understand.

**Reply by jerolomb** [reply]
Thanks for the description, it's not plan for the moment but we will study the possibility
